specs/kiota.config.md

```diff
@@ -0,0 +1,634 @@
+# Kiota Config
+
+Kiota generates client code for an API and stores parameters in a kiota.lock file. A project can contain multiple API clients, but they are independently managed. Kiota has no awareness that an app has a dependency on multiple APIs, even though that is a core use case.
+
+## Status
+
+| Date | Version | Author | Status |
+| -- | -- | -- | -- |
+| November 30th, 2023 | v0.3 | Sébastien Levert | Final Draft |
+| November 22nd, 2023 | v0.2 | Sébastien Levert | Draft |
+| September 24th, 2023 | v0.1 | Darrel Miller | Draft |
+
+## Current Challenges
+
+- Client code generation is not reproducible if API description changes
+- Kiota doesn’t have a good solution for APIs that use multiple security schemes.
```

Comment:
```suggestion
- Kiota doesn’t have an obvious solution for APIs that use multiple security schemes.
```
the current solution is good and works well, if you know the expectation is "generate multiple clients" but since we don't advertise that anywhere, it's hard to come to that conclusion.